### PR TITLE
Recipe for Message ix

### PR DESCRIPTION
**Pull request by gidden**

### CODE REVIEW

File: recipes/message-ix/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+package:
+  name: message-ix
+  version: {{ version }}
+
+source:
+  url: https://github.com/iiasa/message_ix/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (mariusvniekerk) [inline recipes/message-ix/meta.yaml]: Can this package be noarch?

Comment (gidden) [reply recipes/message-ix/meta.yaml]: It should be @mariusvniekerk - added in cf0df8e, should pass CI

**Comment by gidden** [comment]
looks good @mariusvniekerk

File: recipes/message-ix/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{% set version = "1.0.0-prerelease" %}
```
Comment (jakirkham) [inline recipes/message-ix/meta.yaml]: So this version isn't valid. Interestingly this is noted in the CI logs, but doesn't fail the CI. Could you please send a follow-up PR to correct this? Thanks in advance.

ref: https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#package-version

cc @isuruf

Comment (jakirkham) [reply recipes/message-ix/meta.yaml]: Note to others, this is currently blocking conversion at `staged-recipes`.

cc @conda-forge/staged-recipes

**Pull request merged**